package.json

```diff
@@ -28,13 +28,15 @@
     "knex": "^0.13.0",
     "knex-postgis": "^0.2.2",
     "lodash": "^4.17.4",
+    "lodash.uniqby": "^4.7.0",
```

Comment:
Any reason to do this versus just use what's already in the lodash version already included?